roles/openshift_health_checker/openshift_checks/memory_availability.py

```diff
@@ -0,0 +1,33 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class MemoryAvailability(OpenShiftCheck):
+    """Check that recommended memory is available."""
+
+    name = "memory_availability"
+    tags = ["preflight"]
+
+    recommended_memory_gb = {
+        "nodes": 8.0,
+        "masters": 16.0,
+        "etcd": 20.0
```

Comment:
It is good practice to include a comma at the end of every line in multiline definitions like this.
It makes it easier to add / remove / move items without having to make changes to other lines.